lib/x11utils.pm

```diff
@@ -78,12 +79,21 @@ sub ensure_unlocked_desktop {
         my @tags = qw(displaymanager displaymanager-password-prompt generic-desktop screenlock screenlock-password authentication-required-user-settings authentication-required-modify-system guest-disabled-display oh-no-something-has-gone-wrong);
         push(@tags, 'blackscreen') if get_var("DESKTOP") =~ /minimalx|xfce/;    # Only xscreensaver and xfce have a blackscreen as screenlock
         push(@tags, 'gnome-activities') if check_var('DESKTOP', 'gnome');
-        assert_screen \@tags, no_wait => 1;
+        push(@tags, 'gnome-activities') if (!check_var('DESKTOP', 'gnome') && get_var("FIPS_ENABLED") && is_pvm);
+        # For powerVM x11 access in FIPs mode, we can connect it via vnc even in textmode
+        # Add some wait time for powerVM due to performance issue
+        my $timeout = is_pvm ? '120' : '30';
+        assert_screen(\@tags, timeout => $timeout, no_wait => 1);
         # Starting with GNOME 40, upon login, the activities screen is open (assuming the
         # user will want to start something. For openQA, we simply press 'esc' to close
         # it again and really end up on the desktop
         if (match_has_tag('gnome-activities')) {
             send_key 'esc';
+            # Add some more re-try for powerVM due to performance issue
+            if (get_var("FIPS_ENABLED") && is_pvm) {
+                send_key 'esc';
```

Comment:
we exclude the needle for `gnome-activities` afterward, it is just sending one more 'esc' and waiting for still screen, does not enter in this section again, so doesn't look like 'more retries'. is it really needed the extra 'esc' if it isn't, perhaps reword the comment.

Reply:
Good suggestion!
Let me do